### ISSUE TITLE
Reproduction gap on IU-Xray dataset

### ISSUE THREAD

**Issue by YangSuoze** [SKD-HPC/LHR-RFL#2]
Dear authors,

Thank you for releasing the code. I have recently tried to reproduce your results on the IU-Xray dataset, following the official data split and training settings described in your paper and repository.

However, I found that my reproduced results are significantly lower than the numbers reported in the paper. Below is the comparison:
<html><head></head><body>
Metric | Reported in Paper | Reproduced (Mine)

-- | -- | --

BLEU-1 | 0.539 | 0.412

BLEU-2 | 0.380 | 0.274

BLEU-3 | 0.278 | 0.193

BLEU-4 | 0.210 | 0.140

METEOR | 0.223 | 0.156

ROUGE-L | 0.416 | 0.355

</body></html>

This is my Training log (best BLEU-4 epoch):
<img width="672" height="295" alt="Image" src="https://github.com/user-attachments/assets/b71e4c86-23b5-45b6-8b59-20f0e86f9309" />

It would be great if you could provide some guidance to help reproduce the reported numbers more accurately.

**Comment by SKD-HPC** [comment]
Hello! The performance of the LHR-RFL (or traditional RL)  varies significantly under different randon sseeds on small datasets (IU X-Ray). As a matter of fact, the performance of different models without RL varies significantly under different randon sseeds on small datasets (IU X-Ray). One possible approach is to try different random number seeds. I suggest you test it on the MIMIC-CXR dataset.
